### DISCUSSION TITLE
a problem with large coordinate

### DISCUSSION THREAD

**Discussion by wzzxf218** [mozman/ezdxf#977] [Q&A]
when I use a  large coordinate to add circle, I find that the arc could not be fully drawn, as shown in the figure.
![1](https://github.com/mozman/ezdxf/assets/44667373/20508af5-35c7-485b-9ab4-9726bfab41f1)

my code as follows:

```Python
import ezdxf

doc = ezdxf.new('R2010')
msp = doc.modelspace()
msp.add_circle([495799.372499998, 3325910.029999998], 1)
doc.saveas("1.dxf")
```

but when I scales the coordinates to [49579.9372499998, 332591.0029999998], it can be drawn well.
have you ever encountered such situation ?

**Comment by mozman** [comment]
You need a better DXF viewer:

The `drawing` add-on of `ezdxf` has no problems:

![image](https://github.com/mozman/ezdxf/assets/4230966/69c1f5e8-c4ed-4832-8a70-d28f3f748054)

BricsCAD:
![image](https://github.com/mozman/ezdxf/assets/4230966/44339279-e827-4a3f-8bbd-4e9d0e72e0f8)

DWG Trueview:
![image](https://github.com/mozman/ezdxf/assets/4230966/abe29717-ad44-4aed-a754-abcfbb33c27b)
